pkg/operator/ceph/config/defaults.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2018 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package config provides default configurations which Rook will set in Ceph clusters.
+package config
+
+import "github.com/coreos/pkg/capnslog"
+
+// DefaultCentralizedConfigs returns the default configuration options Rook will set in Ceph's
+// centralized config store. If the version of Ceph does not support the centralized config store,
+// these will be set in a shared config file instead.
+func DefaultCentralizedConfigs(rookLogLevel capnslog.LogLevel) *Config {
+	cephLogLevel := logLevelToCephLogLevel(rookLogLevel)
+
+	c := NewConfig()
+	c.Section("global").
```

Comment:
In the spirit of #2534 i'd suggest we not include some of these options. For example, no need for the logging options.

Reply:
I don't think I was present for the discussion about why the debug options are unnecessary. How are we able to raise the log level in Luminous without these?